### PR TITLE
updating docker tag to latest in environment setup docs

### PR DESCRIPTION
**Pull request by ChamathKB**
# Pull request guidelines

Welcome to the 🐸STT project! We are excited to see your interest, and appreciate your support!

This repository is governed by the Contributor Covenant Code of Conduct. For more details, see the [CODE_OF_CONDUCT.md](CODE_OF_CONDUCT.md) file.

In order to make a good pull request, please see our [CONTRIBUTING.rst](CONTRIBUTING.rst) file, in particular make sure you have set-up and run the pre-commit hook to check your changes for code style violations.

Before accepting your pull request, you will be asked to sign a [Contributor License Agreement](https://cla-assistant.io/coqui-ai/STT).

This [Contributor License Agreement](https://cla-assistant.io/coqui-ai/STT):

- Protects you, Coqui, and the users of the code.
- Does not change your rights to use your contributions for any purpose.
- Does not change the license of the 🐸STT project. It just makes the terms of your contribution clearer and lets us know you are OK to contribute.

### CODE REVIEW

File: doc/playbook/ENVIRONMENT.md [hunk 1]
```diff
@@ -175,17 +175,17 @@ You will now see the `ghcr.io/coqui-ai/stt-train` image when you run the command
 ```
 $ docker image ls
 REPOSITORY                               TAG                  IMAGE ID       CREATED          SIZE
-ghcr.io/coqui-ai/stt-train               v0.10.0-alpha.4      d145cb0930ea   37 minutes ago   5.12GB
+ghcr.io/coqui-ai/stt-train               latest      d145cb0930ea   37 minutes ago   5.12GB
```
Comment (wasertech) [inline doc/playbook/ENVIRONMENT.md:178]: You forgot to update the image ID :)

File: doc/playbook/ENVIRONMENT.md [hunk 2]
```diff
@@ -175,17 +175,17 @@ You will now see the `ghcr.io/coqui-ai/stt-train` image when you run the command
...
 ``` -->
 
 ### Testing the image by creating a container and running a script
 
 Now that you have your Docker image pulled down, you can create a _container_ from the image. Here, we're going to create a container and run a simple test to make sure that the image is working correctly.
 
-_Note that you can refer to Docker images by `id` - such as `7cdc0bb1fe2a` in the example above, or by the image's name and `tag`. Here, we will be using the image name and `tag` - ie `ghcr.io/coqui-ai/stt-train:v0.10.0-alpha.4`._
+_Note that you can refer to Docker images by `id` - such as `7cdc0bb1fe2a` in the example above, or by the image's name and `tag`. Here, we will be using the image name and `tag` - ie `ghcr.io/coqui-ai/stt-train:latest`._
```
Comment (wasertech) [inline doc/playbook/ENVIRONMENT.md:185]: Here too.

**Comment by wasertech** [comment]
Otherwise LGTM!

File: doc/playbook/ENVIRONMENT.md (hunk 1 above)
Comment (ChamathKB) [reply doc/playbook/ENVIRONMENT.md]: Oops my bad. Thanks 🙌️

File: doc/playbook/ENVIRONMENT.md (hunk 2 above)
Comment (ChamathKB) [reply doc/playbook/ENVIRONMENT.md]: Updated this too

**Comment by wasertech** [comment]
Merged since it doesn't impact code base.

Thanks for contributing.

**Pull request merged**